src/fsharp/service/IncrementalBuild.fs

```diff
@@ -1022,16 +1023,91 @@ module IncrementalBuilderEventTesting =
 
 module Tc = FSharp.Compiler.TypeChecker
 
+[<AutoOpen>]
```

Comment:
AutoOpen really?  seems a bit broad.

Reply:
I wouldn't say it's broad; there is only one item in the module, which is `SyntaxTree`.